src/routes/statusPage/index.js

```diff
@@ -0,0 +1,187 @@
+    const [categoryList, setCategoryList] = useState([]);
+    const [path, setPath] = useState('');
+
+    const {
+        handleSubmit,
+        register,
+        setValue,
+    } = useForm({});
+
+    const {
+        register: register2,
+        setValue: setValue2,
+        handleSubmit: handleSubmit2,
+      } = useForm({});
+
+    useEffect(()=> {
+		axios.get(`${BASE_URL}/status-page/config/`, {
+			headers: {
+				Authorization:`Token ${getUserToken()}`
+			}
+		}).then((response)=>{
+			setValue("path", response.data["path"])
+            setPath(response.data['path'])
+		})
+    }, [])
+
+    useEffect(()=> {
+		axios.get(`${BASE_URL}/status-page/category/`, {
+			headers: {
+				Authorization:`Token ${getUserToken()}`
+			}
+		}).then((response)=>{
+			setCategoryList(response.data)
+		})
+    }, [categoryList])
+    
+    const onSaveUrl = (data) => {
+        const formData = new FormData()
+        formData.append('path', data.path)
+
+        axios.post(`${BASE_URL}/status-page/config/`, formData, {
+            headers: {
+                'Authorization':`Token ${getUserToken()}`
+            }
+        }).then((response)=>{
+            setErrorMessageSaveUrl('')
+            setPath(response.data['path'])
+        }).catch((error)=> {
+            setErrorMessageSaveUrl(error.response.data.path[0]);
+        })
+        
+    };
+    const onAddCategory = (data) => {
+        const formData = new FormData()
+        formData.append('name', data.name)
+
+        axios.post(`${BASE_URL}/status-page/category/`, data, {
+            headers: {
+                'Authorization':`Token ${getUserToken()}`
+            }
+        }).then((response)=>{
+            categoryList.push(response.data)
+            setValue2('name', '')
+        })
+        
+    };
+
+    return (
+        <div style={{padding: "40px 20px"}}>
+            <Flex
+                h="full"
+                backgroundRepeat='no-repeat'
+                justify='center'
+                align='center'
+            >
+                {<Grid w='60vw' h='full' justify='center' templateColumns='repeat(6, 1fr)'>
+                    <GridItem colSpan={6}>
+                        <Box verticalAlign='center'>
+                            <Text mb={23} fontSize='32px' fontWeight='semibold' color='black'>
+                            Status Page Configuration
+                            </Text>
+                            
+                            <Box mb='30px' />
+
+                            <Text mb={23} fontSize='32px' fontWeight='semibold' color='black'>
+                            Status Page URL
+                            </Text>
+                            
+                            <Box mb='10px' />
+                            <Box as='form' onSubmit={handleSubmit(onSaveUrl)} id='form-save-url'>
+                                <Flex align='center'>
+                                    <Text>https://status.monapi.com/</Text>
```

Comment:
```suggestion
                                    <Text>{window.location.origin}/status/</Text>
```
kayaknya ini bisa dari ini aja

Reply:
done